src/main/java/com/lothrazar/storagenetwork/gui/NetworkWidget.java

```diff
@@ -264,12 +265,6 @@ public void mouseClicked(double mouseX, double mouseY, int mouseButton) {
         return;
       }
     }
-    if (searchBar.mouseClicked(mouseX, mouseY, mouseButton)) {
-      if (mouseButton == UtilTileEntity.MOUSE_BTN_RIGHT) {
```

Comment:
so youre saying " Clearing with RMB still works perfectly." thats really cool im checking this out this weekend